src/components/FolderSection.js

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from "react";
+import { getFolderList, getAllLinks, getFolderLink } from "../api";
+import CardList from "./CardList";
+import SearchBar from "./SearchBar";
+import addImg from "../assets/add.svg";
+import shareIcon from "../assets/share.svg";
+import penIcon from "../assets/pen.svg";
+import deleteIcon from "../assets/delete.svg";
+import "./FolderSection.css";
+
+export default function FolderSection() {
+  const [folderName, setFolderName] = useState("");
+  const [folderList, setFolderList] = useState([]);
+  const [cardList, setCardList] = useState([]);
+
+  //전체 폴더 클릭
+  async function getAllList() {
+    try {
+      const { data } = await getAllLinks();
+      setCardList(data);
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  async function folderAllNameClick(e) {
+    setFolderName(e.target.textContent);
+    await getAllList();
+  }
```

Comment:
UI에 보여져야 하는 텍스트가 변경되면 이 코드는 깨질 것 같은데요~
`전체`를 클릭했을때만 실행되어야 한다면 이렇게 해줘도 괜찮지 않을까요?
```suggestion
  async function folderAllNameClick() {
    setFolderName('전체');
    await getAllList();
  }
```